docs/sphinx/install.rst

```diff
@@ -141,6 +141,41 @@ we refer to the `CUDA Quantum GitHub repository`_.
 
 .. _CUDA Quantum GitHub repository: https://github.com/NVIDIA/cuda-quantum/blob/main/Building.md
 
+
+CUDA Quantum Dependencies
+-------------------------
+
+CUDA Quantum can be used to simulate quantum programs (see `using/simulators`) on a CPU-only system, but a GPU is highly recommended.
+
+The supported CPUs include both the x86_64 (x86-64-v3 when using the OpenMPI stack) and ARM64 architectures.
+
+For simulation of CUDA Quantum programs on NVIDIA GPUs, additional packages should be installed. See also `cuQuantum <https://docs.nvidia.com/cuda/cuquantum/latest/getting_started.html>`__.
+
+The following table summarizes the required components.
+
+.. list-table:: Supported Systems
+    :widths: 30 50
+    :header-rows: 0
+
+    * - CPU architectures
+      - x86_64, ARM64
+    * - Operating System
+      - Linux
+    * - Tested Distributions
+      - Ubuntu 22.04
```

Comment:
Sorry for the slow response, but here's what I had in mind with my prior comment about `.github/workflows/config/validation_config.json`.
```suggestion
    * - Tested Distributions (C++)
      - Ubuntu 22.04
    * - Tested Distributions (Python 3.8)
      - Fedora 38
    * - Tested Distributions (Python 3.9)
      - Fedora 38, Debian 11
    * - Tested Distributions (Python 3.10)
      - Fedora 38, Ubuntu 22.04
    * - Tested Distributions (Python 3.11)
      - Fedora 38, Ubuntu 22.04, Debian 12, Redhat UBI 8.0, Redhat UBI 9.2
```

Reply:
+1. 
Python wheels are available for x86_64 and aarch64 processor architectures. The following Linux distributions are supported:
 
Ubuntu - 22.04
RHEL - 8,9 
Debian - 11,12 
Fedora - 38
CentOS 8 
Rocky - 8,9:

(OpenSUSE/SLES should work but I'll add it to the list once I've added it to the test matrix)

Reply:
To clarify why CentOS and Rocky are listed: Docker images for CentOS are superseded by the redhat ubi that we use for testsing, and similarly "Rocky Linux is intended to be a downstream, complete binary-compatible release using the Red Hat Enterprise Linux (RHEL) operating system source code." - ie. these three should all be covered by the redhat ubi validation.